index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/89d87f2112.js" crossorigin="anonymous"></script>
+  <title>Portafolio</title>
+</head>
+
+<body>
+    <header class="header">
+      <div class="tallbar"></div>
+      <nav class="navbar">
+        <h2 class="logo">My logo</h2>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="title">
+        <h1>I’m Ammar.
+          Glad to see you!</h1>
+      </div>
+      <div class="parrafo">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+          and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      </div>
+      <div class="connect">
+        <img src="img/LET’S CONNECT.png" alt="connection image">
+      </div>
+      <div class="socialMedia">
+        <a href=""><img src="img/social-media.png" alt="social media images"class="icon"></a>
+        <!-- <a href=""><img src="/Images/Linkedin icon.png" alt="" class="icon"></a>
+        <a href=""><img src="/Images/Vector (6).png" alt="" class="icon icon-3"></a>
+        <a href=""><img src="/Images/Vector (4).png" alt="" class="icon"></a>
+        <a href=""><img src="/Images/Vector (5).png" alt="" class="icon"></a> -->
+
+      </div>
+
+    
+      <section>
+
+      </section>
+    </main>
```

Comment:
- [ ] Excellent job designing the HTML layout for your headline section 🚀. However, please kindly make sure that you use the `main` semantic tag to wrap all your sections, currently, you are using the `main` tag as the main wrapper for your headline but it would be a good idea to use a `section` tag for this instead and then include that section inside your `main` tag 🤓.
This is a small example of a semantic HTML design 👇🏽, you can read more information about it [here](https://www.w3schools.com/html/html5_semantic_elements.asp)

```
<html>
    <head>
        <title>Portfolio</title>
    </head>
    <body>
        <header>
            Here goes logo, navigation, etc.
        </header>
        <main>
            <section>
                Here you can include your headline
            </section>
            <section>
                Here you can include My recent works
            </section>
        </main>
        <footer>
            Footer information, links, etc.
        </footer>
    </body>
</html>
```